The build error seems to be some timing error:

```
Exception in thread Thread-1 (most likely raised during interpreter shutdown):
Traceback (most recent call last):
File "/usr/lib/python2.7/threading.py", line 551, in __bootstrap_inner
File "/home/travis/build/OCA/connector/connector/queue/worker.py", line 313, in run
File "/home/travis/build/OCA/connector/connector/queue/worker.py", line 328, in check_alive
File "/usr/lib/python2.7/contextlib.py", line 35, in __exit__
File "/home/travis/build/OCA/connector/connector/session.py", line 81, in session
File "/home/travis/build/OCA/connector/connector/session.py", line 157, in close
File "/home/travis/odoo-7.0/openerp/sql_db.py", line 161, in wrapper
File "/home/travis/odoo-7.0/openerp/sql_db.py", line 290, in close
File "/home/travis/odoo-7.0/openerp/sql_db.py", line 300, in _close
File "/home/travis/odoo-7.0/openerp/sql_db.py", line 262, in print_log
<type 'exceptions.TypeError'>: unsupported operand type(s) for +=: 'NoneType' and 'int'
Exception AttributeError: "'NoneType' object has no attribute 'warning'" in <bound method Cursor.__del__ of <openerp.sql_db.Cursor object at 0x5854c10>> ignored
```

--- a/connector/queue/worker.py
+++ b/connector/queue/worker.py
@@ -19,6 +19,7 @@
 #
 ##############################################################################
 
+import re
 import logging
 import os
 import threading
@@ -265,6 +266,9 @@ class WorkerWatcher(threading.Thread):
                 db_names = services['db'].exp_list(True)
             else:
                 db_names = []
+        dbfilter = config['dbfilter']
+        if dbfilter and db_names:
+            db_names = [d for d in db_names if re.match(dbfilter, d)]
         available_db_names = []
         for db_name in db_names:
             session_hdl = ConnectorSessionHandler(db_name,

